biocrnpyler/global_mechanism.py

```diff
@@ -200,15 +200,20 @@ class Degredation_mRNA_MM(GlobalMechanism, MichaelisMenten):
        ComplexSpecies are seperated by this process, including embedded ComplexSpecies. 
        OrderedPolymerSpecies are ignored.
     """
-    def __init__(self, nuclease, name="rna_degredation_mm", mechanism_type = "rna_degredation", **keywords):
+    def __init__(self, nuclease, name="rna_degredation_mm", mechanism_type = "rna_degredation", 
+        default_on = False, recursive_species_filtering = True, filter_dict = None, **keywords):
```

Comment:
Can we do this by passing **keywords throughout and using particular keyword arguments wherever they are needed (instead of having all keywords in the init call? 

In other words, if all of the keyword arguments make sense in this class's init call, then this is fine.

Reply:
I thought of that, but decided to make all the normal global mechanism keywords visible because I thought it would help inspire people to reuse these mechanisms in different ways.